### PR TITLE
Add note for Office Hours

### PR DESCRIPTION
**Pull request by suhanime**
/assign @dgoodwin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,3 +34,7 @@ In the future Hive will support more cloud providers.
   * [Hive Architecture](./docs/architecture.md)
   * [SyncSet](./docs/syncset.md)
   * [SyncIdentityProvider](./docs/syncidentityprovider.md)
+  
+# Office Hours
+
+Hive team hosts virtual office hours every third Friday at 2 PM EST, open for Hive users. Please reach out for an invite.
```
Comment (dgoodwin) [inline README.md:40]: Can we add a Google meet link? (we wouldn't want to use our normal scrum line but a separate link would be good)

Comment (suhanime) [reply README.md]: Added

**Comment by dgoodwin** [comment]
Sorry :) I realize now people still can't tell when the next meeting is.

I found a publish this event link, can you add in a link with: https://calendar.google.com/event?action=TEMPLATE&tmeid=MzBqcmhoOTJkZjNkcDI4NnZhaTJwcXJhaDBfMjAyMDEyMTFUMTkwMDAwWiBkZ29vZHdpbkByZWRoYXQuY29t&tmsrc=dgoodwin%40redhat.com&scp=ALL

**Comment by suhanime** [comment]
> Sorry :) I realize now people still can't tell when the next meeting is.
> 
> I found a publish this event link, can you add in a link with: https://calendar.google.com/event?action=TEMPLATE&tmeid=MzBqcmhoOTJkZjNkcDI4NnZhaTJwcXJhaDBfMjAyMDEyMTFUMTkwMDAwWiBkZ29vZHdpbkByZWRoYXQuY29t&tmsrc=dgoodwin%40redhat.com&scp=ALL

This is neat! Fixed.

**Comment by dgoodwin** [comment]
Perfect, thanks!

/lgtm

**Pull request merged**